style.css

```diff
@@ -72,3 +68,192 @@ p {
   background-repeat: no-repeat;
   background-position: right top;
 }
+
+.section-1 {
+  margin-top: 40px;
+  padding: 2em;
+  background: #f7f7f9;
+  border: 2px solid #e1e3e8;
+  height: 100%;
+  width: 100%;
+}
+
+.first-card {
+  margin-top: 80px;
+  background: #fff;
+  padding: 1rem;
+  border: 2px solid #e1e3e8;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1);
+  border-radius: 1rem;
+  height: auto;
+  width: 100%;
+}
+
+h1 {
+  font-size: 30px;
+  color: #172b4d;
+}
+
+.details {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  color: #abd4c0;
+}
+
+.card-img {
+  width: 100%;
+}
```

Comment:
[COMPLIMENT] I like how you used responsive units to make your site accommodate multiple screens 👏 🎉 its a great responsive design 😉